app/actions/network.js

```diff
@@ -0,0 +1,29 @@
+import { NETWORK_EXCEPTION } from '../types/message';
+
+export function fetch(type, data) {
```

Comment:
@psimyn 
When you call fetch() without import it, here's what you gonna get:
```
ReferenceError: fetch is not defined
   at fetchTopics (/Users/xxx/public/assets/server.js:28867:11)
   at /Users/xxx/public/assets/server.js:32693:22
   at Array.map (native)
   at preRenderMiddleware (/Users/xxx/public/assets/server.js:32692:11)
   at /Users/xxx/public/assets/server.js:125:42
   at /Users/xxx/public/assets/server.js:24806:6
   at /Users/xxx/public/assets/server.js:22685:12
   at done (/Users/xxx/public/assets/server.js:23167:20)
   at /Users/xxx/public/assets/server.js:23173:8
   at getComponentsForRoute (/Users/xxx/public/assets/server.js:23357:6)
   at /Users/xxx/public/assets/server.js:23412:6
   at /Users/xxx/public/assets/server.js:23172:6
   at Array.forEach (native)
   at mapAsync (/Users/xxx/public/assets/server.js:23171:10)
   at getComponents (/Users/xxx/public/assets/server.js:23411:29)
   at finishEnterHooks (/Users/xxx/public/assets/server.js:22679:36)
   at runTransitionHooks (/Users/xxx/public/assets/server.js:23011:6)
   at runEnterHooks (/Users/xxx/public/assets/server.js:23054:11)
   at /Users/xxx/public/assets/server.js:22672:43
   at runTransitionHooks (/Users/xxx/public/assets/server.js:23011:6)
   at runChangeHooks (/Users/xxx/public/assets/server.js:23071:11)
   at finishMatch (/Users/xxx/public/assets/server.js:22669:42)
```
_Also note this or any "forgot import errors" will not automatically occurred, you'll only receive this after restart of the server, and I think that might be a another bug unrelated with this PR._